src/controllers/createTenantController.ts

```diff
@@ -26,6 +30,10 @@ export default async function createTenantController(req: Request, res: Response
       currency,
     } = req.body;
 
+    const {
+      file,
+    } = req;
```

Comment:
What do you think of the next suggestion?
```suggestion
    const { file } = req;
```